library/src/morpheus-master/sass/modules/tool/announcements/_announcements.scss

```diff
@@ -40,4 +40,16 @@
 			@include sakai_secondary_button();
 		}
 	}
+
+	.grab_cursor {
+		cursor: grab;
+		cursor:-moz-grab;
+		cursor: -webkit-grab;
+	}
+
+	.grabbing_cursor {
+	    cursor: grabbing;
+	    cursor: -webkit-grabbing;
+	    cursor: -moz-grabbing;
+	}
```

Comment:
I'd prefer we define these once in `library/src/morpheus-master/sass/base/_extendables.scss` instead of across the 3 files.

Reply:
Yes, I think it would be better too. I'm testing it now that I know in which file I should apply it.